package.json

```diff
@@ -62,7 +62,7 @@
   },
   "dependencies": {
     "@flybywiresim/api-client": "~0.7.0",
-    "@flybywiresim/react-components": "^0.2.7",
+    "@flybywiresim/react-components": "^0.3.0",
```

Comment:
Should we rather keep the caret or set a specific version?

Reply:
This version was supposed to be `0.2.10`, but was named `0.3.0` by mistake. This will not be a problem if future versions are named correctly.